src/frontend/src/lib/components/ui/Card.svelte

```diff
@@ -4,7 +4,7 @@
 	import CardAmountDescription from '$lib/components/ui/CardAmountDescription.svelte';
 
 	export let noMargin = false;
-	export let testId: string | undefined = undefined;
+	export let testId = 'card';
```

Comment:
Why did those properties changed? Couldn't we keep the default undefined?

Reply:
No particular reason, i thought to use the same way that gix-components do for UI components

But changed to undefined